src/overlays/actors/ovl_En_Syateki_Crow/z_en_syateki_crow.c

```diff
@@ -97,118 +90,120 @@ void EnSyatekiCrow_Init(Actor* thisx, PlayState* play2) {
     Actor_ProcessInitChain(&this->actor, sInitChain);
     SkelAnime_InitFlex(play, &this->skelAnime, &gGuaySkel, &gGuayFlyAnim, this->jointTable, this->morphTable,
                        OBJECT_CROW_LIMB_MAX);
-    Collider_InitJntSph(play, &this->unk_23C);
-    Collider_SetJntSph(play, &this->unk_23C, &this->actor, &sJntSphInit, &this->unk_25C);
-    this->unk_23C.elements->dim.worldSphere.radius = sJntSphInit.elements[0].dim.modelSphere.radius;
+    Collider_InitJntSph(play, &this->collider);
+    Collider_SetJntSph(play, &this->collider, &this->actor, &sJntSphInit, this->colliderElements);
+    this->collider.elements[0].dim.worldSphere.radius = sJntSphInit.elements[0].dim.modelSphere.radius;
     ActorShape_Init(&this->actor.shape, 2000.0f, ActorShadow_DrawCircle, 20.0f);
 
     if ((path == NULL) || (EN_SYATEKI_CROW_GET_NUMBER(&this->actor) >= 0x80)) {
         Actor_MarkForDeath(&this->actor);
         return;
     }
 
-    this->unk_1C8 = Lib_SegmentedToVirtual(path->points);
-    this->unk_1CC = 1;
-    this->unk_1CE = path->count;
-    this->unk_1C4 = 0x14;
-    this->unk_1BC = 0;
-    func_809CA5D4(this);
+    this->pathPoints = Lib_SegmentedToVirtual(path->points);
+    this->currentPoint = 1;
+    this->maxPoint = path->count;
+    this->deathTimer = 20;
+    this->waitTimer = 0;
+    EnSyatekiCrow_SetupWaitForSpawn(this);
 }
 
 void EnSyatekiCrow_Destroy(Actor* thisx, PlayState* play) {
     EnSyatekiCrow* this = THIS;
 
-    Collider_DestroyJntSph(play, &this->unk_23C);
+    Collider_DestroyJntSph(play, &this->collider);
 }
 
-void func_809CA5D4(EnSyatekiCrow* this) {
+void EnSyatekiCrow_SetupWaitForSpawn(EnSyatekiCrow* this) {
     Actor_SetScale(&this->actor, 0.03f);
     this->actor.speedXZ = 0.0f;
     this->actor.gravity = 0.0f;
     this->actor.world = this->actor.home;
     this->actor.prevPos = this->actor.home.pos;
     this->actor.shape.rot = this->actor.world.rot;
-    this->unk_1CC = 1;
+    this->currentPoint = 1;
```

Comment:
I'd lean towards making this `curPointIndex` since you do get temps that are the actual current point as well.